docs/iOSAppium2SetupGuide-README.md

```diff
@@ -0,0 +1,121 @@
+# Appium 2 – iOS Setup
+
+<span style="font-size: larger;">This guide will walk you through the steps to set up Appium 2 for iOS.</span> 
+
+1. Remove previous Appium 1.x installation if any.
+## Install Appium 2
+
+2. Install Appium 2 using the following command:
+- ```npm i -g appium@next```
+
+
+3. Verify XCUITest Driver installation path.
+![XCUITest-driver-path.png](XCUITest-driver-path.png)
+
+## Install Plugins
+
+
+**Appium Device Farm** is a plugin for Appium that allows to run iOS automation tests on a variety of real devices and simulators. This can be helpful for ensuring that the app works correctly on a variety of devices and iOS versions.
+
+
+**Appium Dashboard plugin*** is a tool that allows to view and interact with iOS automation tests in real time. This can be helpful for debugging and troubleshooting tests.
+
+
+
+**Appium relaxed-caps plugin** is required because it allows to use capabilities that are not prefixed with the vendor's name. This can be helpful for compatibility reasons, as some vendors may not support the vendor prefix for certain capabilities.
+
+
+
+4. To install the Appium Device Farm and Appium Dashboard plugins, run the following commands.
+
+* `appium plugin install --source=npm appium-device-farm`
+
+* `appium plugin install --source=npm appium-dashboard`
+
+
+
+5. Install plugin for Relaxing Appium's requirement for vendor prefixes on capabilities.
+
+* `appium plugin install relaxed-caps`
+
+* `appium plugin install --source=npm @appium/relaxed-caps-plugin`
+
+
+
+## Install Drivers
+
+
+
+6. Cross Platform installation, [click here](https://github.com/danielpaulus/go-ios), this is an operating system independent implementation of iOS device features. You can run UI tests, launch or kill apps, install apps etc. with it.
+
+
+7. The XCUITest driver is used to automate iOS apps that are written using Xcode. To install the driver, run the following command:
+
+* `appium driver install xcuitest`
+
+
+
+## Install and configure Appium Inspector
+
+
+8. Latest [Appium Inspector](https://github.com/appium/appium-inspector/releases) supporting Appium 2
+
+
+9. Appium Inspector [Capability guidelines and examples](https://appium.io/docs/en/2.0/guides/caps/)
+
+
+
+## Start Appium
+
+
+
+10. To start Appium, run the following command:
+
+* `appium --relaxed-security --allow-cors --use-plugins="appium-dashboard,relaxed-caps,device-farm" -pa /wd/hub --plugin-device-farm-platform=both`
```

Comment:
I am guessing this is to start appium manually. Let's make that explicit. Also highlight that teswiz will take care of this automatically

Reply:
Remark added.